tests/utils/testing.rs

```diff
@@ -50,7 +209,10 @@ impl Drop for AybServer {
     }
 }
 
-pub async fn snapshot_storage(db_type: &str) -> Result<SnapshotStorage, AybError> {
-    let config = read_config(&PathBuf::from(server_config_path(db_type)))?;
+pub async fn snapshot_storage(test_type: &str) -> Result<SnapshotStorage, AybError> {
+    let config_path = generate_test_config(test_type).map_err(|e| AybError::Other {
```

Comment:
Do we need to generate the config again? Can we pass the path to this function instead?